### PR TITLE
chore: improve error messages for Helm commands

### PR DESCRIPTION
**Pull request by cnvergence**
Fixes: https://github.com/cilium/cilium-cli/issues/1907

### CODE REVIEW

File: cli/clustermesh.go [hunk 1]
```diff
@@ -243,7 +243,7 @@ func newCmdClusterMeshEnableWithHelm() *cobra.Command {
 			params.HelmReleaseName = helmReleaseName
 			ctx := context.Background()
 			if err := clustermesh.EnableWithHelm(ctx, k8sClient, params); err != nil {
-				fatalf("Unable to enable ClusterMesh: %s", err)
+				fatalf("Unable to enable ClusterMesh: %s, please ensure that you have specified the --namespace and --helm-release-name if you are working with a cluster with customized Cilium installation.", err)
```
Comment (derailed) [inline cli/clustermesh.go:246]: Might be an easier to grok if these read something like `failed to enable/disable xxx`

**Review by derailed — COMMENTED** [review]
@cnvergence Thank you for these updates!

File: cli/clustermesh.go (hunk 1 above)
Comment (cnvergence) [reply cli/clustermesh.go:246]: good point, also I did mistake by starting it with uppercase

Comment (cnvergence) [reply cli/clustermesh.go:246]: I will also include other error messages

Comment (derailed) [reply cli/clustermesh.go]: fatalf is just a wrapper to print a message to stderr and exit. Thus messages can start with upper case.

Comment (cnvergence) [reply cli/clustermesh.go]: correct, I restored it

**Comment by cnvergence** [comment]
@derailed PTAL again :)

**Review by derailed — APPROVED** [review]
@cnvergence Thank you for the updates!

**Comment by cnvergence** [comment]
hi @derailed, shall I fix something in this PR?

**Pull request closed without merging**